README.md

```diff
@@ -984,6 +984,14 @@ in inheritance.
   is a hybrid of `Array`'s intuitive inter-operation facilities and
   `Hash`'s fast lookup.
 
+* Prefer `map` over `collect`.
+
+* Prefer `detect` over `find`.
```

Comment:
i still think the ActiveRecord explanation is necessary here, especially since "find" is more common in Ruby (and also is used in JS).

The rest are idiomatic ruby, so I don't think they need justification.